src/ConformalTracking.cc

```diff
@@ -1106,23 +1095,20 @@ void ConformalTracking::buildNewTracks(std::vector<KDTrack*>& conformalTracks, s
         break;
 
       // Produce all tracks leading back to the seed hit from this cell
-      std::vector<cellularTrack*> candidateTracks;
-      std::vector<cellularTrack*> candidateTracksTemp;
+      UniqueCellularTracks candidateTracks;
+      UniqueCellularTracks candidateTracksTemp;
       createTracksNew(candidateTracksTemp, cells[itCell],
                       usedCells2);  // Move back to using used cells here? With low chi2/ndof?
 
-      for (size_t itTrack = 0; itTrack < candidateTracksTemp.size(); itTrack++) {
-        if (int(candidateTracksTemp[itTrack]->size()) >= (m_minClustersOnTrack - 1))
-          candidateTracks.push_back(candidateTracksTemp[itTrack]);
-        else
-          delete candidateTracksTemp[itTrack];
-      }
+      copy_if(std::make_move_iterator(candidateTracksTemp.begin()), std::make_move_iterator(candidateTracksTemp.end()),
+              std::back_inserter(candidateTracks),
+              [this](UcellularTrack const& track) { return (int(track->size()) >= (m_minClustersOnTrack - 1)); });
```

Comment:
WTF?!

Reply:
That is a strange way to spell `beautiful` :)